modules/src/ics02_client/height.rs

```diff
@@ -119,19 +119,3 @@ impl std::fmt::Display for Height {
         )
     }
 }
-
```

Comment:
FYI -- These two methods have moved to `modules/src/ics24_host/identifier.rs` (see below).